packages/ethereum-contracts/contracts/agreements/gdav1/SuperfluidPool.sol

```diff
@@ -459,10 +456,23 @@ contract SuperfluidPool is ISuperfluidPool, BeaconProxiable {
         emit MemberUnitsUpdated(superToken, memberAddr, oldUnits, newUnits);
     }
 
-    function _claimAll(address memberAddr, uint32 time) internal returns (int256 amount) {
-        amount = getClaimable(memberAddr, time);
+    function _settle(address memberAddr, int256 amount) internal {
```

Comment:
I will have to reject this impelemtation, for design reason:

This function is effectful and has no  constraint on what "amount" can be provided to it. Because of this, now as a code reader, you will have to worry if all the call sites of this function provided this "amount" correctly. The more robust design is to avoid such possibility, keep reasoning of correctness local.

In this case, it is quite easy to do, I will do a change in the following commmit.